code/modules/memestation/arenabonfire.dm

```diff
@@ -0,0 +1,132 @@
+/obj/structure/arenabonfire
```

Comment:
Any reason this isn't a subtype of `/obj/structure/bonfire`?

Reply:
Because I didn't want to be able to add grills or the ability to put it out. arenabonfires don't need air, either